src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingCook.java

```diff
@@ -123,19 +122,8 @@ public void initTagPositions()
             return;
         }
 
-        final IBlueprintDataProvider te = getTileEntity();
-        if (te != null)
-        {
-            initTags = true;
-            sitPositions = new ArrayList<>();
-            for (final Map.Entry<BlockPos, List<String>> entry : te.getWorldTagPosMap().entrySet())
-            {
-                if (entry.getValue().contains(TAG_SITTING))
-                {
-                    sitPositions.add(entry.getKey());
-                }
-            }
-        }
+        sitPositions = getLocationsFromTag(TAG_SITTING);
+        initTags = !sitPositions.isEmpty();
```

Comment:
This is a slight semantic change, but it didn't seem unreasonable.  Alternative is to just set `initTags = true` unconditionally (which is also slightly different).